biojava-structure/src/main/java/org/biojava/nbio/structure/PDBHeader.java

```diff
@@ -589,9 +599,16 @@ public String getTitle() {
 	public void setTitle(String title) {
 		this.title = title;
 	}
+	
+	/**@deprecated will be removed later. Use {@link #getKeywords()}
+	 * @return
+	 */
```

Comment:
Please use the `@Deprecated` code tag as well, see https://stackoverflow.com/questions/9031832/how-to-declare-or-mark-a-java-method-as-deprecated

Reply:
I didn't know that the code tag is part of the language while the javadoc is not. thanks for the info.
Done